packages/create-svelte/shared/+typescript/package.json

```diff
@@ -4,9 +4,9 @@
 		"check:watch": "svelte-check --tsconfig ./tsconfig.json --watch"
 	},
 	"devDependencies": {
-		"typescript": "^4.4.3",
+		"typescript": "^4.5.4",
```

Comment:
~4.5.4 instead?

Reply:
No, this is for the `devDependencies` of the template, not those of the Kit repo, and I don't want to update this every time a new minor version for TS is released.

Reply:
After the discussion in Discord I changed my mind, being more defensive and do `~` makes sense, maybe there are future TS updates that don't play nice with things in Vite/ESBuild.